src/utils/errors.ts

```diff
@@ -3,14 +3,20 @@ import { parseJSONRPCError } from './parsing'
 export const EVM_ERROR_CODES = [
   // JSON RPC Standard errors
   '-32',
+  // Execution reverted error
+  '-3',
```

Comment:
doesn't this automatically include the line above?

Reply:
now as it checks for exact matches

Reply:
Yes, kind of a coincidence though.

Reply:
> no as it checks for exact matches

Why not? It's `.includes()`

Reply:
The error should be 3 and not -3 btw

Reply:
> The error should be 3 and not -3 btw

oh that's why yes